robotbase/src/main/java/com/team2357/frc2022/commands/IntakeSequenceCommandGroup.java

```diff
@@ -0,0 +1,30 @@
+package com.team2357.frc2022.commands;
+
+import com.team2357.frc2022.Constants;
+import com.team2357.frc2022.subsystems.IntakeSubsystem;
+
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import edu.wpi.first.wpilibj2.command.WaitCommand;
+import edu.wpi.first.wpilibj.DoubleSolenoid.Value;
+
+
+public class IntakeSequenceCommandGroup extends SequentialCommandGroup{
+private IntakeSubsystem m_intakeSub;
+
+public IntakeSequenceCommandGroup(IntakeSubsystem intakeSub){
+    m_intakeSub = intakeSub; 
+    addRequirements(m_intakeSub);
+
+if (m_intakeSub.getPivot() == Value.kReverse){
+    addCommands(new IntakeSetPivotCommand(m_intakeSub, Value.kForward), new WaitCommand(0), new IntakeRollerCommand(m_intakeSub, Constants.INTAKE.FORWARD_SPEED));
```

Comment:
Add the use of your PIVOT_WAIT_SECONDS constant here

Reply:
fixed